### DISCUSSION TITLE
Dont Start by location: ^4.4.0

### DISCUSSION THREAD

**Discussion by Arnold-Papplic** [Lyokone/flutterlocation#840] [General]
Hello, 
After updating Gradle and Flutter, I get an error so I can't launch the app. I use location: ^4.4.0 and the error message is : 
* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':location' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.20

**Comment by xdkaka** [comment]
+1
